sdk/iot/common/inc/az_iot_common_internal.h

```diff
@@ -32,6 +32,16 @@
  */
 AZ_NODISCARD int32_t _az_iot_u32toa_size(uint32_t number);
 
+/**
+ * @brief Copies the url-encoded content of `source` span into `destination`, returning the free remaining of `destination`.
+ *
+ * @param[in] destination The span where the url-encoded content is copied to.
+ * @param[in] source The span to url-encode and copy the content from.
```

Comment:
@param[in] source The span to url-encode

_you use "copy" too much_

Reply:
Sorry.